test/functional/DDR_Test/src/j9vm/test/ddrext/Constants.java

```diff
@@ -257,18 +257,18 @@
 	public static final String SHRC_JITHSTATS_FAILURE_KEY = "no shared cache,JITPROFILE data length 0 metadata 0";
 
 	public static final String SHRC_FINDJITH = "findjith";
-	public static final String SHRC_FINDJITH_METHODNAME = "run";
-	public static final String SHRC_FINDJITH_SUCCESS_KEY = "run\\(\\)Ljava/lang/.*,JITHINT data !j9x,.*/Object";
+	public static final String SHRC_FINDJITH_METHODNAME = "<init>";
+	public static final String SHRC_FINDJITH_SUCCESS_KEY = "<init>\\(.*\\)";
```

Comment:
Why was `JITHINT data !j9x,.*/Object` removed here and in other success keys below?

Reply:
I added them back now, please take a look @keithc-ca

Reply:
Ready for review, please take a look @keithc-ca